### DISCUSSION TITLE
puppeteer in thorium

### DISCUSSION THREAD

**Discussion by gaf2419** [Alex313031/thorium#59] [General]
hello, can someone tell me how to make puppeteer use this version of Thorium instead of chromium.

**Comment by Alex313031** [comment]
@gaf2419 See PUPPETEER_SKIP_CHROMIUM_DOWNLOAD in https://pptr.dev/#environment-variables

as well as

https://pptr.dev/api/puppeteer.puppeteernode.product

and

https://pptr.dev/api/puppeteer.puppeteernode.executablepath

I dont use puppeteer but this might help.
